docs/TVM_EP.md

```diff
@@ -83,20 +84,27 @@ export PYTHONPATH=<path_to_onnx_runtime>/build/<OS_NAME>/Release/_deps/tvm-src/p
 ## Configuration options
 TVM Executor Provider can be configured with the following provider options:
 ```python
-po = [dict(target=client_target,
+po = [dict(executor=tvm_executor_type,
+           so_folder=folder_with_pretuned_files,
+           target=client_target,
            target_host=client_target_host,
            opt_level=client_opt_level,
            freeze_weights=freeze,
+           to_nhwc=layout_transform,
+           tuning_type=tvm_optimizer_type,
            tuning_file_path=client_tuning_logfile,
            input_names = input_names_str,
            input_shapes = input_shapes_str)]
 tvm_session = onnxruntime.InferenceSession(model_path, providers=["TvmExecutionProvider"], provider_options=po)
 ```
 <br>
 
+- `executor` is executor type used by TVM. There is choice between two types: GraphExecutor and VirtualMachine which are corresponded to "graph" and "vm" tags. VirtualMachine is used by default.
```

Comment:
Is it possible to know more about the two options or maybe link to another place giving more details unless this choice does not have significant impact.